### DISCUSSION TITLE
v0.1.0-test-1

### DISCUSSION THREAD

**Discussion by trungnt2910** [AzureAms/TimetableApp.Uno#11] [General]
Windows 10 (and 11) package for `TimetableApp.Uno`.

The web version can always be accessed [here](https://AzureAms.github.io/TimetableApp.Uno).

This is a prerelease build, some bugs may occur. You can always find support by opening an Issue.

Notice for Windows package:
- The Windows package (`.msixbundle`) file requires installing a certificate from us (the `.cer` file), before it can be installed successfully. Packages published here can be fully trusted and is always built from the latest source code from the `master` branch. 

<hr /><em>This discussion was created from the release <a href='https://github.com/AzureAms/TimetableApp.Uno/releases/tag/'>v0.1.0-test-1</a>.</em>

**Comment by baorng** [comment]
https://user-images.githubusercontent.com/52161682/132348413-41485a9c-9d5d-4cef-9949-d707853aedb9.mp4

The app crashes as soon as it is launched on my device.

**Reply by trungnt2910** [reply]
Don't know what's the cause, let me try to run that package on my Windows 11 VM.
If that doesn't work, then it might be because of your old Windows version?
Is it the latest 21H1?

**Reply by baorng** [reply]
Yes I'm on 21H1.

**Reply by trungnt2910** [reply]
It's bad, I've reproduced this problem on a Windows 11 VM. I'll try to debug this thing as soon as possible.

**Reply by trungnt2910** [reply]
I'd say, the bug does not have anything to do with the environment, as I've reproduces the bug in both a VM and my own dev machine. It might be some settings-related bug, that only occurs in Release builds.

**Reply by trungnt2910** [reply]
It's not a JSON bug LMAO. It's some obscure bug by Microsoft. I'll fix it soon.
